recipes/scala/src/main/scala/com/adobe/platform/ml/ScoringDataLoader.scala

```diff
@@ -47,15 +49,17 @@ class ScoringDataLoader extends DataLoader {
     val serviceToken: String = sparkSession.sparkContext.getConf.get("ML_FRAMEWORK_IMS_ML_TOKEN", "").toString
     val userToken: String = sparkSession.sparkContext.getConf.get("ML_FRAMEWORK_IMS_TOKEN", "").toString
     val orgId: String = sparkSession.sparkContext.getConf.get("ML_FRAMEWORK_IMS_ORG_ID", "").toString
-
-    val dataSetId: String = configProperties.get("scoringDataSetId").getOrElse("")
+    val timeframe: String = configProperties.get("timeframe").getOrElse("")
```

Comment:
I dont see the timeframe in the pipeline.json file. Can you add it for an example?

Reply:
Yes, I'll add it for an example.